### PR TITLE
enh: improve dust global agent

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

- switch to sonnet 3.5
- give it websearch capabiliies
- add more prompting

## Risk

tested on our workspace with a custom assistant, but we can't be sure there won't be any regressions.

## Deploy Plan

N/A

### CODE REVIEW

File: front/lib/api/assistant/global_agents.ts [hunk 1]
```diff
@@ -857,9 +867,29 @@ async function _getDustGlobalAgent(
     versionAuthorId: null,
     name,
     description,
-    instructions:
-      "Assist the user based on the retrieved data from their workspace." +
-      `\n${BREVITY_PROMPT}`,
+    instructions: `<info>
+The assistant answers the user's questions.
```
Comment (spolu) [inline front/lib/api/assistant/global_agents.ts:871]: Too verbose and too overfitted on a leaked set of instructions that may not even be the truth :)

Comment (spolu) [reply front/lib/api/assistant/global_agents.ts:871]: The most important is:
If we believe in the very high level instructions the they should be added to the meta prompt for Sonnet. But I doubt they serve a big purpose.

If you really really believe in using the tag <instruction> then you should likely build it for all users when using Claude models.

Comment (fontanierh) [reply front/lib/api/assistant/global_agents.ts]: Ok made them shorter, simpler and less over-fitted. 
I'll add a claude-specific set of "tools-use meta instructions".

**Review by philipperolet — APPROVED** [review]
LGTM

**Pull request merged**